libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -134,6 +134,15 @@ const decoratePromo = (elem, index) => {
   const isImageOnly = elem.matches('.image-only');
   const imageElem = elem.querySelector('picture');
 
+  if (!isImageOnly) {
+    const imageElemMaybe = imageElem && elem.removeChild(imageElem.parentElement.parentElement);
```

Comment:
Chaining `.parentElement` properties could be problematic if anything changes in the structure. How about something like
```
const cachedImageElem = imageElem && elem.removeChild(imageElem.closest(`${selectors.gnavPromo} > div`));
```